src/main/resources/swagger/api-docs.yaml

```diff
@@ -136,6 +136,38 @@ paths:
             - email
             - profile
             - https://www.googleapis.com/auth/devstorage.full_control
+  /api/inputsOutputs:
+    post:
+      tags:
+        - Method Repository
+      operationId: getMethodIO
+      summary: Get information about a method's inputs and outputs
+      produces:
+        - application/json
+      parameters:
+        - name: methodName
+          description: name of Method to look up
+          required: true
+          type: string
+          in: body
+          schema:
+            $ref: MethodID
```

Comment:
not your doing originally, but go down to the definition for MethodId around line 2142, and add default: 0 to the methodVersion. Without the default value, swagger won't show this key when pre-filling the payload. (with methodVersion missing from the payload, swagger returns no response; that was my problem)